src/components/SecondaryPanes/Breakpoints/Breakpoint.js

```diff
@@ -63,6 +63,13 @@ class Breakpoint extends PureComponent<Props> {
     showContextMenu({ ...this.props, contextMenuEvent: e });
   };
 
+  onDoubleClick = () => {
+    const { breakpoint, openConditionalPanel } = this.props;
+    if (breakpoint.condition) {
+      openConditionalPanel(breakpoint.location.line);
```

Comment:
hmm, what happens if the breakpoint is for a different file than the currently selected file?

Reply:
It will open and select the file that is associated with the breakpoint.  Luckily this just worked without me having to change anything.